src/NewRelic/JsonNewRelic.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace App\NewRelic;
+
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class JsonNewRelic
+{
+    private HttpClientInterface $newRelicClient;
+    private int $appId;
+    private string $Relickey;   //. config
+    private string $RelicHost; ///
```

Comment:
```suggestion
    private string $apiKey;
    private string $host;
```
On va garder comme ça finalement, c'est pas une mauvaise façon de faire.